src/components/auth/LoginContainer.tsx

```diff
@@ -132,6 +209,17 @@ export const LoginContainer: React.FC<LoginContainerProps & LoginContainerDispat
         onCancelLogin={handleCancelLogin}
         onClose={handleCloseSelector}
       />
+
+      <EthWalletNewSelector
+        open={showWalletSelector.open && isSignInFlowV3}
+        loading={loading}
+        availableProviders={availableProviders || defaultAvailableProviders}
+        provider={providerInUse}
+        onLogin={handleLogin}
+        canceling={canceling}
+        onCancelLogin={handleCancelLogin}
+        onClose={handleCloseSelector}
+      />
```

Comment:
These changes are being made to perform an A/B test, so we try to keep the components well separated, using the same names but adding the word "New" to identify them. In case the tests are successful, the change of components will be easier. When applying this type of changes and using a single component (which we could do the same with the rest of the components that are being duplicated as LoginItemContainer) would be more difficult to read the files and slower refactoring at the end of the A/B tests.